frontend/src/pages/RecurringRunDetailsV2.test.tsx

```diff
@@ -0,0 +1,469 @@
+/*
+ * Copyright 2018 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render, screen, waitFor } from '@testing-library/react';
+import * as React from 'react';
+import { CommonTestWrapper } from 'src/TestWrapper';
+import RecurringRunDetailsV2 from './RecurringRunDetailsV2';
+import TestUtils from 'src/TestUtils';
+import { V2beta1RecurringRun, V2beta1RecurringRunStatus } from 'src/apisv2beta1/recurringrun';
+import { Apis } from 'src/lib/Apis';
+import { PageProps } from './Page';
+import { RouteParams, RoutePage, QUERY_PARAMS } from 'src/components/Router';
+import { shallow, ReactWrapper, ShallowWrapper } from 'enzyme';
+import { ButtonKeys } from 'src/lib/Buttons';
+
+describe('RecurringRunDetailsV2', () => {
+  let tree: ReactWrapper<any> | ShallowWrapper<any>;
```

Comment:
Prefer react testing library over enzyme.

Reply:
Will create a backlog item for this implementation.